### ISSUE TITLE
Job - possible long earn possibilities are not fair weighted

### ISSUE THREAD

**Issue by Tattooo** [D3nnis3n/White-Tiger-Feedback#84]
Hi,

i am a mid active player on white tiger, so i have everyday 1-2 hours of spending and enjoying the game. So please respect the situation i try to describe as an issue from this point of view - i know that the game experience differs a lot if you are able to spent +8 hours.

I play some rounds now on white tiger, so im not a complete newb, but also not following all changes every now and then.

I feel like i need to pick mining. As i did this time. I can make money from day 1-30 with a stable time -> income rate - not changing a lot over all of the time. As mashines get better ratios the prices drop a bit - but that is compensated.

I do not see a lot of other Jobs that offer that kind of stable income possibility - so it is sort of mandantory to keep up my house, my mashines, my food to a level where i can pick a second jobs which i maybe enjoy or want to try. There are other good designed jobs which have a long life in the economy, but they are way less profitable as mining (lumberjack, gatherer, farmer).

Especially as gaining stars is so slow at the server - and i understand the circumstances - it hinders me as a lower time invest player into enjoying some jobs.

My suggestion would be:
Give me a choice, keep the actual system for players who would like to cross jobs from several "main" speccs.
Allow me to take the option to only become a cook - and allow me to pick at an easier rate gatherer and or farmer with camp fire cook (just an example) - but force me to stay in that job selection so only backing, cooking, advanced cooking... beeing available for my second star.

With this suggestion not all jobs need to be reworked to have similar profit gain ratio as mining, but it would open up the possibility to stay a bit more sustainable with some more jobs.

I this time wanted to become a Glassworker - and realized how low gain this Job is up till now. Glass is nearly unsellable at the moment it becomes available, the requirements demand a high invest - and as glass only sells at 3 Euro, a compacted iron at 6 euro - it is much easier to go into my iron mine and earn money from mining.

To sum up
I do not think mining should be nurfed, but some other jobs could be paired up to offer a similar level of joy - and income. On a long term, of course it would be favourable if all jobs are on a similar level - but until then, such a mechanik could ease a bit the pain.

Thanks for reading, recognizing and thinking about the idea - keep on doing the good work :)
